package.json

```
@@ -24,8 +24,8 @@
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn constraints --fix",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!**/CHANGELOG.md' '!**/CHANGELOG.old.md' '**/*.yml' '!.yarnrc.yml' '!merged-packages/**' --ignore-path .gitignore",
     "prepack": "./scripts/prepack.sh",
-    "prepare-preview-builds": "yarn workspaces foreach --parallel run prepare-manifest:preview",
-    "publish-previews": "yarn workspaces foreach --parallel run publish:preview",
+    "prepare-preview-builds": "scripts/prepare-preview-builds.sh",
```

Nit: Everywhere else we seem to use `./` for relative paths. Not entirely sure what the consequences are for omitting it (it might have platform-specific challenges)
```suggestion
    "prepare-preview-builds": "./scripts/prepare-preview-builds.sh",
```